src/Domain/Messenger/MessengerGateway.php

```diff
@@ -35,8 +35,35 @@ class MessengerGateway extends QueryableGateway
 
     private static $tableName = 'gibbonMessenger';
     private static $primaryKey = 'gibbonMessengerID';
-    private static $searchableColumns = [''];
+    private static $searchableColumns = ['gibbonMessenger.subject', 'gibbonMessenger.body'];
     
+    /**
+     * Queries the list of users for the Manage Users page.
```

Comment:
Looks like a reused doc block. Maybe worth noting that this does not determine which messages to display on, say, the message wall, but is for managing messages.